dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -598,7 +598,7 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 						</GridItem>
 					</StandFirstGrid>
 				</ElementContainer>
-				{showKeyEventsCarousel && CAPIArticle.keyEvents.length && (
+				{showKeyEventsCarousel && CAPIArticle.keyEvents.length > 0 ? (
```

Comment:
Am I right that _technically_ making this a comparison means it will return a boolean so we don't need, need the ternary? @mxdvl @OllysCoding 

Either way, having it here means we're solid